packages/wct-mocha/src/config.ts

```diff
@@ -93,11 +93,13 @@ export function get<K extends keyof Config>(key: K): Config[K] {
   return _config[key];
 }
 
-export function deepMerge(target: Partial<Config>, source: Partial<Config>) {
-  Object.keys(source).forEach((key) => {
-    if (target[key] !== null && typeof target[key] === 'object' &&
-        !Array.isArray(target[key])) {
-      deepMerge(target[key], source[key]);
+export function deepMerge<V extends {}>(target: V, source: V) {
```

Comment:
😲I am learning some TypeScript from this PR.

Reply:
Yeah, I'd prefer to rewrite these deep merge functions (we have two!) into something less magical.. but that's for another time